playwright/e2e/modules/custom-component.spec.ts

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
Bias towards saying this spec reasonably covers the coverage needed by SonarCloud, but can add unit tests as well if needed?